rest-heroes/src/main/resources/application-azure.yml

```diff
@@ -0,0 +1,7 @@
+"%azure":
```

Comment:
If the filename is `application-azure.yml` then I don't think you need the `%azure` key...

Reply:
Correct. I just tested it